cli/demo_connect_four.sh

```diff
@@ -0,0 +1,160 @@
+#!/bin/bash
+
+# setup:
+# run ajuna node
+#   target/release/ajuna  --dev --tmp --ws-port <NODEPORT> --port 30385 --rpc-port <RPCPORT>
+#
+# run worker
+#   rm light_client_db.bin
+#   rm -r shards
+#   export RUST_LOG=integritee_service=info,ita_stf=debug
+#   integritee-service init_shard
+#   integritee-service shielding-key
+#   integritee-service signing-key
+#   integritee-service -P <WORKERPORT> -p <NODEPORT> -r 3485 run --dev --skip-ra
+#
+# then run this script
+
+# usage:
+#  export RUST_LOG_LOG=integritee-cli=info,ita_stf=info
+#  demo_connect_four.sh -p <NODEPORT> -P <WORKERPORT> -m file
+#
+# if -m file is set, the mrenclave will be read from file
+
+while getopts ":m:p:P:" opt; do
+    case $opt in
+        m)
+            READMRENCLAVE=$OPTARG
+            ;;
+        p)
+            NPORT=$OPTARG
+            ;;
+        P)
+            RPORT=$OPTARG
+            ;;
+    esac
+done
+
+# using default port if none given as arguments
+NPORT=${NPORT:-9944}
+RPORT=${RPORT:-2000}
+
+echo "Using node-port ${NPORT}"
+echo "Using trusted-worker-port ${RPORT}"
+
+AMOUNTSHIELD=1000
```

Comment:
That's not a shielding anymore, but rather the starting balance (or similar) of the accounts.